Bricks/file/file.h

```diff
@@ -54,12 +54,17 @@ namespace current {
 // Platform-indepenent, injection-friendly filesystem wrapper.
 struct FileSystem {
 #ifndef CURRENT_WINDOWS
-  static constexpr char PathSeparatingSlash = '/';
+  static constexpr char PathSeparator = '/';
```

Comment:
Because in broader communities the concept this name designates is called "path separator", and nowhere it is called "path separating _anything_". I gave links to a couple examples in the respective commit message.